### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by olawalejarvis**
### What does this PR do?
Implement all feedback gotten from Adebayo

### CODE REVIEW

File: server/app/controllers/Document.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    */
+  getDocument(req, res) {
+    db.Document
+      .findById(req.params.id)
+      .then((document) => {
+        if (!document) {
+          return res.status(404)
+            .send({
+              success: false,
+              message: 'This document cannot be found'
```
Comment (xmile1) [inline server/app/controllers/Document.js]: you can do this at the middleware level or use a helper function

Comment (olawalejarvis) [reply server/app/controllers/Document.js]: Okay
Thanks
Done

File: server/app/middlewares/Auth.js [hunk 2]
```diff
@@ -0,0 +1,600 @@
...
+  /**
+   * Query for search terms
+   * @param {Array} terms array of search terms
+   * @returns {Object} return user's data
+   */
+  likeSearch(terms) {
+    const like = {
+      $or:
+      [
+        { title: { $ilike: { $any: terms } } },
```
Comment (xmile1) [inline server/app/middlewares/Auth.js]: does this work, is it not supposed to be` $iLike` as opposed to `$ilike`. `L` being uppercase

Comment (olawalejarvis) [reply server/app/middlewares/Auth.js]: Oh, Yes, it works 
Thanks.
Done

**Pull request merged**